src/main/scala/scala/tools/partest/sbt/SBTRunner.scala

```diff
@@ -6,12 +6,15 @@
 **                          |/                                          **
 \*                                                                      */
 
-package scala.tools.partest
-package nest
+package scala.tools.partest.sbt
```

Comment:
It's used by `PartestTask` in the same package. This used to be in `partest-interface` before we moved it into partest itself.